### PR TITLE
Added a full suite of ancient surgical tools.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
## Description of changes
Adds a set of surgical tools based loosely on this page: https://exhibits.hsl.virginia.edu/romansurgical/index.html
Sadly I could not find a game scenario where 20 varieties of speculum would be worthwhile.

## Why and what will this PR improve
Adds some more thematically appropriate surgical tools for the fantasy map.

## Authorship
Myself.

## Changelog
:cl:
add: Simple surgical tools can now be crafted.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/surgery_tools.dm [hunk 1]
```diff
@@ -52,7 +52,7 @@
  */
 /obj/item/cautery
 	name = "cautery"
-	desc = "This stops bleeding."
+	desc = "A surgical tool for preventing bleeding or closing incisions."
```
Comment (out-of-phaze) [inline code/game/objects/items/weapons/surgery_tools.dm]: technically i think it stops bleeding, it doesn't prevent it? i mean i guess it prevents further bleeding

File: code/modules/crafting/stack_recipes/recipes_textiles.dm [hunk 2]
```diff
@@ -84,3 +84,10 @@
 /decl/stack_recipe/textiles/fur/bedding
 	result_type           = /obj/item/bedsheet/furs
 	category              = "bedding"
+
+/decl/stack_recipe/textiles/surgical_sutures
+	result_type           = /obj/item/ancient_surgery/sutures
+	craft_stack_types     = list(/obj/item/stack/material/thread)
+	difficulty            = MAT_VALUE_HARD_DIY
+	category              = "medical"
+	available_to_map_tech_level = MAP_TECH_LEVEL_MEDIEVAL
```
Comment (out-of-phaze) [inline code/modules/crafting/stack_recipes/recipes_textiles.dm:93]: spacing here doesn't match up but i don't care for this style anyway, so not blocking

**Review by out-of-phaze — DISMISSED** [review]

File: code/game/objects/items/weapons/surgery_tools.dm (hunk 1 above)
Comment (MistakeNot4892) [reply code/game/objects/items/weapons/surgery_tools.dm]: `halting` maybe?

**Review by out-of-phaze — DISMISSED** [review]

File: code/game/objects/items/weapons/surgery_tools_ancient.dm [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+/obj/item/ancient_surgery/sutures/get_tool_properties()
+	var/static/list/tool_properties = list(TOOL_SUTURES = TOOL_QUALITY_MEDIOCRE)
+	return tool_properties
+
+/obj/item/ancient_surgery/bonesaw
+	name = "bonesaw"
+	desc = "A heavy saw for cutting through bones."
+	icon = 'icons/obj/items/surgery/bonesaw.dmi'
+	material = /decl/material/solid/organic/meat/gut
```
Comment (comma) [inline code/game/objects/items/weapons/surgery_tools_ancient.dm]: gutsaw?

Comment (MistakeNot4892) [reply code/game/objects/items/weapons/surgery_tools_ancient.dm]: Huh that's meant to be on the sutures, oops

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**